src/main/scala/cognite/spark/v1/DefaultSource.scala

```diff
@@ -121,6 +122,15 @@ class DefaultSource
     )(sqlContext)
   }
 
+  private def createWellDataLayer(
+      parameters: Map[String, String],
+      config: RelationConfig,
+      sqlContext: SQLContext
+  ): WellDataLayerRelation = {
+    val model = parameters.getOrElse("wdlDataType", sys.error("wdlDataType must be specified"))
```

Comment:
prior to use `CdfSparkException("'viewSpaceExternalId' id of the 'space' must be specified")` rather than sys.error 😅

Reply:
BTW, see lines 112 and 114 just above this function.
This is where we got it. ;)